README.md

```diff
@@ -37,15 +37,37 @@ npm test
 
 ## Usage
 
-### Add/Update a Verifiable Credential
-
-Please add or update credentials in the `./credenitals` dir. When adding new
-credential, make sure the directory name is camel-cased. The directory should
-contain a logo image (with `.jpg` or `.png` file extensions) and the credential
-in the `credential.json` file. Additionally, you can also add a file for the
-credential context in the `./contexts` dir (for example see -
-`./contexts/alumni/v1.json`) if you want to host the contexts in
-[`@credential-handler/vc-playground-contexts-server`](https://github.com/credential-handler/vc-playground-contexts-server).
+### Add / Update a Verifiable Credential
+
+To add or update credentials in the `./credentials` directory:
+
+1. Within the `./credentials` directory, create a new directory for the
+credential. Make sure to use camel case for the directory name.
```

Comment:
#15 started to use [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case).  The 'introToChapi' dir doesn't follow that pattern.

Reply:
Updated [a707276](https://github.com/credential-handler/vc-examples/pull/16/commits/a707276374dde6b5fadc0ec65d84587cdc9c15d4) and fixed docs [40aaaa2](https://github.com/credential-handler/vc-examples/pull/15/commits/40aaaa29ff59686dcfb3f8336092f8661e6eab42)